utils/send-mail.js

```diff
@@ -0,0 +1,11 @@
+
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SEND_GRID_API_KEY);
```

Comment:
read in the environment variable in `/app/config/index.js` as `email.apiKey` and then import the config object